Samples/excel-content-add-in/src/dropdown.js

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+/**
+ * Dropdown Plugin
+ *
+ * Given .ms-Dropdown containers with generic <select> elements inside, this plugin hides the original
+ * dropdown and creates a new "fake" dropdown that can more easily be styled across browsers.
+ *
+ * @param  {jQuery Object}  One or more .ms-Dropdown containers, each with a dropdown (.ms-Dropdown-select).
+ * @return {jQuery Object}  The same containers (allows for chaining).
```

Comment:
seems like we shouldn't be using jQuery in this file...? can we convert to use plain JS?